src/v2/Apps/Collect/Routes/Collection/Components/CollectionsHubRails/ArtistSeriesRail/__tests__/ArtistSeriesRail.jest.tsx

```diff
@@ -88,19 +88,23 @@ describe.skip("ArtistSeriesRail", () => {
       const component = getWrapper()
       component.find("a").at(2).simulate("click")
 
-      expect(trackEvent).toBeCalledWith({
-        action: "clickedArtistSeriesGroup",
-        context_module: "artistSeriesRail",
-        context_page_owner_id: "1234",
-        context_page_owner_slug: "slug",
-        context_page_owner_type: "collection",
-        curation_boost: false,
-        destination_page_owner_id: undefined,
-        destination_page_owner_slug: undefined,
-        destination_page_owner_type: "artistSeries",
-        horizontal_slide_position: 2,
-        type: "thumbnail",
-      })
+      expect(trackEvent.mock.calls[0]).toMatchInlineSnapshot(`
+            Array [
+              Object {
+                "action": "clickedArtistSeriesGroup",
+                "context_module": "context-module",
+                "context_page_owner_id": "context-page-owner-id",
+                "context_page_owner_slug": "context-page-owner-slug",
```

Comment:
hm, this is surprising to me - shouldn't the snapshot have matched the old test expectations (line 91 - 102 on the left)? Especially since we explicitly set the ID, slug, and page owner type on lines 60 - 62 above. Is there something I'm missing here @gkartalis ?

Reply:
great catch! So in this case maybe I should revert everything since the tests are marked with a `skip` thus this test file will never run.

Reply:
oh yeah...huh. I mean, seems like maybe we should either a) remove the test suite or b) fix it, since skipped tests are so icky. definitely would expand the scope of this PR though if we choose to fix it. I'll leave it up to you!

Reply:
Cool, I will completely delete them and will take it on me to re-write the tests in another pr (goes straight into my #ff list)